etc/env/example.env

```diff
@@ -14,7 +14,7 @@ CANDIG_AUTH_DOMAIN=candig.docker.internal
 CANDIG_SITE_LOCATION=UHN
 CANDIG_DEBUG_MODE=1
 CANDIG_PRODUCTION_MODE=0
-CANDIG_VERSION=v4.1.0
```

Comment:
I thought we normally wait to update the versions when we do stable releases, this version doesn't actually exist yet, same for the katsu and ingest versions

Reply:
ok, will roll back